error: linking with `cc` failed: exit status: 1

I am getting this error

<details><summary>Error</summary>
<p>

```
cargo r --release 
   Compiling gpui_test v0.1.0 (/home/chakra/Projects/rust/gpui_test)
warning: unused `Result` that must be used
  --> src/main.rs:27:9
   |
27 | /         cx.open_window(
28 | |             WindowOptions {
29 | |                 window_bounds: Some(WindowBounds::Windowed(bounds)),
30 | |                 ..Default::default()
...  |
36 | |             },
37 | |         );
   | |_________^
   |
   = note: this `Result` may be an `Err` variant, which should be handled
   = note: `#[warn(unused_must_use)]` on by default
help: use `let _ = ...` to ignore the resulting value
   |
27 |         let _ = cx.open_window(
   |         +++++++

error: linking with `cc` failed: exit status: 1
  |
  = note: LC_ALL="C" PATH="/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin:/home/chakra/.cargo/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/sbin:/usr/sbin" VSLANG="1033" "cc" "-m64" "/tmp/rustcf3Jy53/symbols.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.00.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.01.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.02.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.03.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.04.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.05.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.06.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.07.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.08.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.09.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.10.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.11.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.12.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.13.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.14.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.gpui_test.4605b1e102214242-cgu.15.rcgu.o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea.nmal2z5qnywu5te.rcgu.o" "-Wl,--as-needed" "-L" "/home/chakra/Projects/rust/gpui_test/target/release/deps" "-L" "/usr/lib/x86_64-linux-gnu" "-L" "/home/chakra/Projects/rust/gpui_test/target/release/build/curl-sys-3197d5c8a5236571/out/build" "-L" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bstatic" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libgpui-8353d268b363b1bc.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libx11_clipboard-194ee18bd435e0ac.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/liboo7-196103d16a6832dc.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libpbkdf2-b4cf4876d60d074b.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libaes-7ca33dae22b4c22b.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcbc-5513b43c6352dbc3.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnum_bigint_dig-f0e275d97b541f56.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnum-5359093e5223054b.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnum_iter-937673976f4be765.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnum_complex-34ec231be17a97e8.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libmd5-f743f9b3073bf628.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libhkdf-b43cb12732aa9a53.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libhmac-894d0dadf843533f.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcipher-deeab3d2e312e8cf.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libinout-38ef43ec77e13422.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libblock_padding-ce4e04ce81c89278.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libzeroize-f9c20bf4cba60ce9.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfont_kit-19ec42c36b31c10f.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfloat_ord-5d974138e5dc4555.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libdirs_next-93af4a04d15023ce.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libdirs_sys_next-28edc13f11828a2a.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfontconfig_sys-b237d89134968c86.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libconst_cstr-34ff18458af6bbb2.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfreetype-bccf93488774567b.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfreetype_sys-9152ec48e99ecef5.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/liblazy_static-f84e7b85a33efd5c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnum_cpus-fc2332c0c3befc64.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/liblinkme-38ba530c755e68aa.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libschemars-39a023d130045a58.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libdyn_clone-6565d947c465e754.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libresvg-33cc66d356adfff9.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libusvg-c391933b946a7042.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libbase64-bf144e0217b93dab.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libdata_url-5e42efde7248c298.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libimagesize-6bb8c565461eeef4.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libxmlwriter-4c9645086907c35d.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsimplecss-b85e07355a0c9c9b.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsvgtypes-13849e270df9698c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libkurbo-4da7acdef429b0b4.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsiphasher-27f54e0735453eff.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtiny_skia-fe5240d0a9e986fc.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtiny_skia_path-27964498829a3f0a.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libstrict_num-967f66f282d08707.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfloat_cmp-7ed8d7bcd8afc7be.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libarrayref-e49e96baa3c671b9.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsemantic_version-571c8ba86dc5c073.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libseahash-1b361f844d10e46b.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libetagere-c82d09b776a129e3.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsvg_fmt-9cc396a387682416.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libeuclid-f198beb3e766f0b2.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libblade_util-9cf095abd5cf279b.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libxim-2ede0e843ca1d392.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libahash-3c5c52bb26d09703.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libconst_random-6a49ce28ad4dea41.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libzerocopy-a620e5d56a52f18e.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libxim_ctext-505398a273d43661.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libxim_parser-26d3cabf434346f7.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libx11rb-946425492540eca6.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libx11rb_protocol-304b90432ac6c88f.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libblade_graphics-7614b241acc87af6.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libgpu_alloc-05243741f0b42deb.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libash_window-6709818111dbe597.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libraw_window_handle-1aa3bf20c4ff6d52.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libmint-a5edca15eb817b4b.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libgpu_alloc_ash-abacfd9a5b1a2eda.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libgpu_alloc_types-1eb3a52e8780f398.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libash-ed3b4614310d0745.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnaga-a82332c6af8ad0aa.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libhexf_parse-615721c299f6e42a.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libunicode_xid-39bc584ba6e887ad.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libbit_set-f82ed9518ba64eed.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libbit_vec-5811b77ac0a9d7c8.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcodespan_reporting-4efe5f63634a6ced.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libunicode_width-917c62174d4905c6.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtermcolor-140cbf3424e89596.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libspirv-fbceedf67378b920.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libuuid-6b5f4a9f0d3d24ef.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsha1_smol-80d8e4eddc137c39.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libwayland_cursor-9e4937e2ba97c5d4.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libxcursor-72632e95d7226a86.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libwayland_protocols_plasma-ee70df4abb3a88d0.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcalloop_wayland_source-e2b1d735cead602c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libxkbcommon-3641a02d6ed3ca2e.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libxkeysym-04ec724116d9de7c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libas_raw_xcb_connection-3ad6739c1f96c12e.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libwayland_protocols-1c91bf6d3c5fb8c2.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libwayland_client-569c50494a50ec5d.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libwayland_backend-f7c1a6c8c7f15964.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libdowncast_rs-ca56a142da6cf33e.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libscoped_tls-cd3e85c798570ecb.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libwayland_sys-3637a5b95ac66fb1.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libdlib-1637521cd6d21872.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/liblibloading-25c852932d37394e.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfiledescriptor-9da3b3980a36c7ae.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libashpd-d537fb7068130e79.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_net-d1176a421b33676b.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libzbus-a7c7b4c0b0560b3c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_fs-58dd616694c78aea.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libhex-af252f55c7db2850.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libxdg_home-a6cc6b7283e1e989.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsha1-3e8bee96060e3b5e.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libordered_stream-3474931b33878f40.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_broadcast-07955663b85e08c9.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnix-cf0bde6bd362ae34.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libmemoffset-0426e9530894b5e1.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libzbus_names-cde00c022df0f7e4.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libzvariant-483ea3f4a10dd44e.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libenumflags2-8dccf090a5e32aee.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libendi-cd3ec18df22bf402.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libstatic_assertions-621e4189fa8de7f2.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcalloop-21ea3d632f8dc743.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libpathfinder_geometry-3dd9102cd5800c78.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libpathfinder_simd-cc08f479ca4c96db.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libitertools-b55eaf6d10616cbc.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcosmic_text-227de66ad5906d56.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libunicode_linebreak-0d459c519091a052.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsys_locale-e169a9c369825842.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libswash-f7a024593b7f9d71.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libyazi-2de276c646896d40.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libskrifa-5911ecec99603279.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libread_fonts-68bfff0869ef0c39.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfont_types-52f3994ec797531a.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libzeno-1aea4f931b80fe37.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libself_cell-211e12851c19bf98.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librustybuzz-65c142f25c54c4e3.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libunicode_bidi_mirroring-dbbb446c0460dbb3.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libunicode_ccc-a574b324fd0222d0.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libunicode_properties-86a4cc1698dcae55.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libunicode_script-eb75dc6045451db7.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libunicode_segmentation-0bf3ecdc222e1b5a.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfontdb-511d2a89180c04b1.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfontconfig_parser-2d0b711c174f53f3.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libroxmltree-8a4f5db97e4977a9.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libmemmap2-9a4abba4ee8513ca.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libttf_parser-ccb3996ee069c466.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librangemap-b4addf87ea3b4f14.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsum_tree-e18fd41af8ef2bda.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librefineable-ba27279a6d5c4911.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtaffy-d97bb45cb644d60c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libgrid-0f4d8e5e582b3867.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libimage-acfdad48651158d3.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libimage_webp-856f6423dd41c206.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libbyteorder_lite-e9ce254ad98ce87d.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libqoi-743b0753da54ba56.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libpng-c3dfc080aabda144.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfdeflate-17899bcd46bd8403.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libexr-4464150ee2caac99.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libflume-0f0374168b84e8fe.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnanorand-50145e6b54d74313.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libspin-b6356977d8a12e09.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libzune_inflate-7e89b31fa2190b3d.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libbit_field-7f0b83cd1700e19f.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/liblebe-d22a3dba8ba44b55.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libhalf-38802717f164f5b6.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libzune_jpeg-8fcdd29259a78a47.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libzune_core-018dd9e876c9ee80.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libgif-40659211bc1b7842.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcolor_quant-ab535074a3996a0c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libbyteorder-aa0457883c2438dd.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libravif-998eef5595b2cd99.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libloop9-a27195a6877b1238.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libavif_serialize-1fd3d653c08f0ecf.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libquick_error-edf39d37239799a7.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librgb-16390ed226c6a734.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librav1e-a4aa15ef07c6b657.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libprofiling-4f4561db07cf6e18.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libav1_grain-9241f597bbdafbe2.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnom-453f4df54a5f0711.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnum_rational-550bc7ffa24fae98.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnum_bigint-f3de7782a4be4edd.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnum_integer-fbf43879f5d00657.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libdebug_unreachable-dcd88d5fa82621eb.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libmaybe_rayon-a44731d021d0562a.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librayon-c9cdc3d040c90dff.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librayon_core-0d6132e3a8d709c6.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcrossbeam_deque-3209635f82d54eff.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcrossbeam_epoch-59eea47378ab31d0.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libv_frame-84feea784515e0b5.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libaligned_vec-d4980423dec3cc28.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libarrayvec-19b51ce57c665252.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libthiserror-104f55a0d4bc251d.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libbitstream_io-80fda9b86769927c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libitertools-b5a4c6eda271dd8c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libeither-e768b7ba901b9c9a.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libimgref-e30af4dfe85749cd.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libbytemuck-cf4ab599182fef7c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnum_traits-5bd2ddc02162f470.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtiff-099c4f77bb2043eb.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libflate2-4e9ad7c1181967ae.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libminiz_oxide-3fa0d650747c11c0.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsimd_adler32-e055fe1695157870.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcrc32fast-bbf4468674ecb6a2.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libweezl-759fdace01e38832.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libjpeg_decoder-d55b0b3c9e00ae8a.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libparking_lot-72993a7c5ae338a4.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libparking_lot_core-38f0d1e4bff2edcf.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsmallvec-974dc126535f184f.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/liblock_api-9dcdd5c07dbfa153.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libscopeguard-8278c67d28bf9eb1.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libutil-18f32c56d57446aa.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librust_embed-aacda1bbe7194f45.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librust_embed_utils-d754a23be5d555bb.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libwalkdir-08fbe456e0f4a372.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsame_file-9bf1eaa85ad1aa05.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsha2-11e40883f1b060bb.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcpufeatures-a623c320f5e97d33.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libdigest-f5f680ab9f54ff23.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsubtle-bd986c448a3a4288.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libblock_buffer-0ad9b16c6900340c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcrypto_common-de265b097ef48091.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libgeneric_array-6dbe66bd5b0b3cc5.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtypenum-fbed62ee51d7486d.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libdirs-a371b844c9aea6cc.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libdirs_sys-8ee4a9cea603afe3.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtake_until-4e1f1fe3d958296a.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libunicase-c2bd655922c113df.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libregex-3aa79d8c27ffc4fe.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librand-8c70f70be755b381.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librand_chacha-0b6f148c917e7ff0.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libppv_lite86-4e9edc39c341aaf1.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librand_core-3d68911c841cb755.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libgetrandom-7bf5762fbf9367db.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libglobset-576d6fbb27f23554.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libregex_automata-eeb69741379cd54f.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libregex_syntax-f685b407d43a7e83.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libbstr-5eefd22a822e9ac6.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libaho_corasick-c4914f0505dba4e2.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libhttp-ecdf3a3ecfde4feb.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libisahc-166151396e46f6f2.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libmime-17df9bd1b635e51e.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcastaway-93bc6fa65373e2e0.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libencoding_rs-8bb76540613a7f51.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtracing_futures-a427cf1ba99fca5a.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libpin_project-23b01c2ca1c000aa.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libhttp-90812e95dfb739b0.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libbytes-7e0b00d421064aad.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfnv-9b2107b1faf780b2.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsluice-380c8aaf2c8b8a02.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcurl-513d3169bc949ba8.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libopenssl_probe-ac1ad1fd708eb02a.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsocket2-50408afc2d794267.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcurl_sys-5c372d023540f7aa.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libopenssl_sys-2acab3985b139b4d.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/liblibz_sys-39221326693d90d9.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/liburl-699375654afe4fe7.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libidna-25ee92b75f746580.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libunicode_normalization-4c1f3965a6e7f178.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtinyvec-ca9ff18c9c230ac0.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtinyvec_macros-a83e724f6958d0b5.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libunicode_bidi-57be1554aa560acb.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libform_urlencoded-a50dc3b5e4ba8825.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libpercent_encoding-8b0347f7b5f16c9b.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtime-272ce04d4e526e1c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtime_core-18e7f591500fa74d.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libnum_conv-f0a9a4a148a0cb6f.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libderanged-0debd3af3e17c707.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libpowerfmt-f946c78eeb781044.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsmol-fd3fa112d67e3b0f.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_process-a71c6c6abb28271f.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_signal-ecaefc2d814d1b98.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsignal_hook_registry-00ab306394da23b5.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_io-87d545e0f06cd5b5.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libpolling-fef074768539b8ee.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtracing-0d8b79437e945f88.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtracing_core-4cb5edca2c4e3a51.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libonce_cell-09fe0f2ed9be70e2.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_lock-800744698bdb512a.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librustix-942580cec28f31c4.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libbitflags-e61005f8faea4380.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/liblinux_raw_sys-d010424454a896a0.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libevent_listener-4d0ce654668a696d.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_net-0acaf6fd27a7f217.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_fs-5445702f31b6dc12.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_channel-19ac030ba0d05768.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libblocking-c871ec49054fa411.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libpiper-e65f2e231d695414.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libatomic_waker-397d504a2d60aa3b.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_channel-8346268dc7f15ae8.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libevent_listener_strategy-6f8094d9f08db783.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libevent_listener-703fdc3f2624278b.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_io-363a7a81e4b630a3.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librustix-adbca5c232877cc0.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libio_lifetimes-c568d0b29d3409ce.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libbitflags-c9f49548e3c37de9.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/liblinux_raw_sys-bf92867ad7ba01e9.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libpolling-08c121e95efeda0c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/liblog-b38ae688a8f00991.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libvalue_bag-a841d7cd535f0c5d.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libvalue_bag_serde1-cb994c07b1edf834.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libserde_fmt-a7a1e60f1e1860d0.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/liberased_serde-d4a5406a7f3c63b0.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libtypeid-85ac27bbebbaf660.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcfg_if-22ef46fe85ecc493.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_lock-f284bef245662e5e.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libevent_listener-33ec9d4fac6caf4c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libsocket2-3f5da35290faff2c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/liblibc-96359edc24c3557f.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfutures_lite-39de7644af3755ca.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfastrand-61faad6457c39a1c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libwaker_fn-9927b9b95e672f9c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_executor-aa72d11a01299fc2.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfutures_lite-370733c17699bb1e.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libparking-b38bfd16cfc94a5c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfastrand-8d0ac4de36e8220a.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libconcurrent_queue-f29c07b0b108d058.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcrossbeam_utils-a7f5acfdd83951e9.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libasync_task-22625a1234192622.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libslotmap-12967ddb52d765ab.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfutures-ce3e2e6b709cec01.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfutures_executor-af83820f5f58626c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfutures_util-a8fb18ebb456d276.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libmemchr-7b7c8fe1df9e1d7c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfutures_io-c95a1d69e390d8b5.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libslab-c70456cab102ec58.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfutures_channel-2dfc7ded8cd1c671.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libpin_project_lite-350ac90dc81bb19e.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfutures_sink-0af1cd03f440741d.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfutures_task-caa26bf5c5e442c2.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libpin_utils-368731c7ed9dab98.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libfutures_core-d08117c1b05cc3eb.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libserde_json-de25c845d7413239.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libitoa-62d5f500fa4013d4.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libryu-82befac54e813a2c.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libindexmap-69dd91b53e0ac604.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libequivalent-3e6ecf7ab8d4a815.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libhashbrown-c87414f7126d2bd8.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libserde-db5ef4b282b22eda.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libcollections-60047954ce2b3ac8.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/librustc_hash-64f78dac3943ed5f.rlib" "/home/chakra/Projects/rust/gpui_test/target/release/deps/libanyhow-683791536abda366.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-d5189b81a4fa4d36.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libpanic_unwind-6a4779412a873200.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libobject-c88c426dd6780435.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libmemchr-e74540b31113a555.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libaddr2line-8751b61bd13c15cf.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libgimli-a8b99dba9f449259.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_demangle-f0068d76172a0372.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd_detect-c16dfaf47799564e.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libhashbrown-bfa26dd63e299db5.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_alloc-75a2330a693e738f.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libminiz_oxide-76779dce1f7ab63d.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libadler-01c5cc588623cb35.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libunwind-07bb9745ec737292.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcfg_if-6eacdcc91004cefb.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liblibc-ae2488b58226c836.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liballoc-52acaddcaaba04c6.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_core-326b78eac9ecd050.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcore-307ebf19f0f13d30.rlib" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-d9076ee5964191bf.rlib" "-Wl,-Bdynamic" "-lfreetype" "-lxcb" "-lxkbcommon" "-lxkbcommon-x11" "-ldl" "-lssl" "-lcrypto" "-lz" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc" "-Wl,--eh-frame-hdr" "-Wl,-z,noexecstack" "-L" "/home/chakra/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-o" "/home/chakra/Projects/rust/gpui_test/target/release/deps/gpui_test-c7a3041a986d9dea" "-Wl,--gc-sections" "-pie" "-Wl,-z,relro,-z,now" "-Wl,-O1" "-Wl,--strip-debug" "-nodefaultlibs" "-fuse-ld=lld"
  = note: ld.lld: error: undefined symbol: __start_linkm2___GPUI_ACTIONS
          >>> referenced by gpui.cae18563361f206d-cgu.09
          >>>               gpui-8353d268b363b1bc.gpui.cae18563361f206d-cgu.09.rcgu.o:(_$LT$gpui..action..ActionRegistry$u20$as$u20$core..default..Default$GT$::default::hf96fbe2a5ac69350) in archive /home/chakra/Projects/rust/gpui_test/target/release/deps/libgpui-8353d268b363b1bc.rlib
          >>> referenced by gpui.cae18563361f206d-cgu.09
          >>>               gpui-8353d268b363b1bc.gpui.cae18563361f206d-cgu.09.rcgu.o:(gpui::action::__GPUI_ACTIONS::he107e90d63d21f0d) in archive /home/chakra/Projects/rust/gpui_test/target/release/deps/libgpui-8353d268b363b1bc.rlib
          >>> the encapsulation symbol needs to be retained under --gc-sections properly; consider -z nostart-stop-gc (see https://lld.llvm.org/ELF/start-stop-gc)
          
          ld.lld: error: undefined symbol: __stop_linkm2___GPUI_ACTIONS
          >>> referenced by gpui.cae18563361f206d-cgu.09
          >>>               gpui-8353d268b363b1bc.gpui.cae18563361f206d-cgu.09.rcgu.o:(_$LT$gpui..action..ActionRegistry$u20$as$u20$core..default..Default$GT$::default::hf96fbe2a5ac69350) in archive /home/chakra/Projects/rust/gpui_test/target/release/deps/libgpui-8353d268b363b1bc.rlib
          >>> referenced by gpui.cae18563361f206d-cgu.09
          >>>               gpui-8353d268b363b1bc.gpui.cae18563361f206d-cgu.09.rcgu.o:(gpui::action::__GPUI_ACTIONS::he107e90d63d21f0d) in archive /home/chakra/Projects/rust/gpui_test/target/release/deps/libgpui-8353d268b363b1bc.rlib
          
          ld.lld: error: undefined symbol: __stop_linkme___GPUI_ACTIONS
          >>> referenced by gpui.cae18563361f206d-cgu.09
          >>>               gpui-8353d268b363b1bc.gpui.cae18563361f206d-cgu.09.rcgu.o:(_$LT$gpui..action..ActionRegistry$u20$as$u20$core..default..Default$GT$::default::hf96fbe2a5ac69350) in archive /home/chakra/Projects/rust/gpui_test/target/release/deps/libgpui-8353d268b363b1bc.rlib
          >>> referenced by gpui.cae18563361f206d-cgu.09
          >>>               gpui-8353d268b363b1bc.gpui.cae18563361f206d-cgu.09.rcgu.o:(gpui::action::__GPUI_ACTIONS::he107e90d63d21f0d) in archive /home/chakra/Projects/rust/gpui_test/target/release/deps/libgpui-8353d268b363b1bc.rlib
          
          ld.lld: error: undefined symbol: __start_linkme___GPUI_ACTIONS
          >>> referenced by gpui.cae18563361f206d-cgu.09
          >>>               gpui-8353d268b363b1bc.gpui.cae18563361f206d-cgu.09.rcgu.o:(_$LT$gpui..action..ActionRegistry$u20$as$u20$core..default..Default$GT$::default::hf96fbe2a5ac69350) in archive /home/chakra/Projects/rust/gpui_test/target/release/deps/libgpui-8353d268b363b1bc.rlib
          >>> referenced by gpui.cae18563361f206d-cgu.09
          >>>               gpui-8353d268b363b1bc.gpui.cae18563361f206d-cgu.09.rcgu.o:(_$LT$gpui..action..ActionRegistry$u20$as$u20$core..default..Default$GT$::default::hf96fbe2a5ac69350) in archive /home/chakra/Projects/rust/gpui_test/target/release/deps/libgpui-8353d268b363b1bc.rlib
          >>> referenced by gpui.cae18563361f206d-cgu.09
          >>>               gpui-8353d268b363b1bc.gpui.cae18563361f206d-cgu.09.rcgu.o:(gpui::action::__GPUI_ACTIONS::he107e90d63d21f0d) in archive /home/chakra/Projects/rust/gpui_test/target/release/deps/libgpui-8353d268b363b1bc.rlib
          >>> the encapsulation symbol needs to be retained under --gc-sections properly; consider -z nostart-stop-gc (see https://lld.llvm.org/ELF/start-stop-gc)
          collect2: error: ld returned 1 exit status
          

warning: `gpui_test` (bin "gpui_test") generated 1 warning
error: could not compile `gpui_test` (bin "gpui_test") due to 1 previous error; 1 warning emitted
```

</p>
</details>

<details><summary>System info</summary>
<p>

```
System:
  Kernel: 6.9.9-1-liquorix-amd64 [6.9-8~mx23ahs] arch: x86_64 bits: 64 compiler: gcc v: 12.2.0 parameters: audit=0
    intel_pstate=disable BOOT_IMAGE=/vmlinuz-6.9.9-1-liquorix-amd64 root=UUID=<filter> ro quiet
    splash
  Desktop: Xfce v: 4.18.1 tk: Gtk v: 3.24.36 info: xfce4-panel wm: xfwm v: 4.18.0 vt: 7
    dm: LightDM v: 1.26.0 Distro: MX-23.3_ahs_x64 Libretto January 21  2024 base: Debian GNU/Linux
    12 (bookworm)
Machine:
  Type: Desktop Mobo: Micro-Star model: MAG B550 TOMAHAWK MAX WIFI (MS-7C91) v: 1.0
    serial: <superuser required> UEFI: American Megatrends LLC. v: 2.60 date: 10/10/2023
CPU:
  Info: model: AMD Ryzen 9 5950X bits: 64 type: MT MCP arch: Zen 3+ gen: 4 level: v3 note: check
    built: 2022 process: TSMC n6 (7nm) family: 0x19 (25) model-id: 0x21 (33) stepping: 2
    microcode: 0xA20120E
  Topology: cpus: 1x cores: 16 tpc: 2 threads: 32 smt: enabled cache: L1: 1024 KiB desc: d-16x32
    KiB; i-16x32 KiB L2: 8 MiB desc: 16x512 KiB L3: 64 MiB desc: 2x32 MiB
  Speed (MHz): avg: 2205 high: 2371 min/max: 2200/5083 boost: enabled scaling:
    driver: acpi-cpufreq governor: ondemand cores: 1: 2200 2: 2200 3: 2200 4: 2371 5: 2200 6: 2200
    7: 2200 8: 2200 9: 2200 10: 2200 11: 2200 12: 2200 13: 2200 14: 2200 15: 2200 16: 2200 17: 2200
    18: 2200 19: 2200 20: 2200 21: 2200 22: 2200 23: 2200 24: 2200 25: 2200 26: 2200 27: 2200
    28: 2200 29: 2200 30: 2200 31: 2200 32: 2200 bogomips: 217610
  Flags: avx avx2 ht lm nx pae sse sse2 sse3 sse4_1 sse4_2 sse4a ssse3 svm
  Vulnerabilities:
  Type: gather_data_sampling status: Not affected
  Type: itlb_multihit status: Not affected
  Type: l1tf status: Not affected
  Type: mds status: Not affected
  Type: meltdown status: Not affected
  Type: mmio_stale_data status: Not affected
  Type: reg_file_data_sampling status: Not affected
  Type: retbleed status: Not affected
  Type: spec_rstack_overflow mitigation: Safe RET
  Type: spec_store_bypass mitigation: Speculative Store Bypass disabled via prctl
  Type: spectre_v1 mitigation: usercopy/swapgs barriers and __user pointer sanitization
  Type: spectre_v2 mitigation: Retpolines; IBPB: conditional; IBRS_FW; STIBP: always-on; RSB
    filling; PBRSB-eIBRS: Not affected; BHI: Not affected
  Type: srbds status: Not affected
  Type: tsx_async_abort status: Not affected
Graphics:
  Device-1: NVIDIA GA104 [GeForce RTX 3060] vendor: Micro-Star MSI driver: nvidia v: 535.183.01
    non-free: 530.xx+ status: current (as of 2023-03) arch: Ampere code: GAxxx process: TSMC n7 (7nm)
    built: 2020-22 pcie: gen: 4 speed: 16 GT/s lanes: 16 bus-ID: 2b:00.0 chip-ID: 10de:2487
    class-ID: 0300
  Display: x11 server: X.Org v: 1.21.1.7 compositor: xfwm v: 4.18.0 driver: X: loaded: nvidia
    unloaded: fbdev,modesetting,nouveau,vesa alternate: nv gpu: nvidia display-ID: :0.0 screens: 1
  Screen-1: 0 s-res: 2560x1440 s-dpi: 96 s-size: 677x381mm (26.65x15.00") s-diag: 777mm (30.58")
  Monitor-1: DP-0 res: 2560x1440 hz: 165 dpi: 108 size: 600x330mm (23.62x12.99")
    diag: 685mm (26.96") modes: N/A
  API: OpenGL v: 4.6.0 NVIDIA 535.183.01 renderer: NVIDIA GeForce RTX 3060/PCIe/SSE2
    direct-render: Yes
Audio:
  Device-1: NVIDIA GA104 High Definition Audio vendor: Micro-Star MSI driver: snd_hda_intel
    v: kernel pcie: gen: 4 speed: 16 GT/s lanes: 16 bus-ID: 2b:00.1 chip-ID: 10de:228b class-ID: 0403
  Device-2: AMD Starship/Matisse HD Audio vendor: Micro-Star MSI driver: snd_hda_intel v: kernel
    pcie: gen: 4 speed: 16 GT/s lanes: 16 bus-ID: 2d:00.4 chip-ID: 1022:1487 class-ID: 0403
  API: ALSA v: k6.9.9-1-liquorix-amd64 status: kernel-api tools: alsamixer,amixer
  Server-1: PipeWire v: 1.0.0 status: active with: 1: pipewire-pulse status: active
    2: wireplumber status: active 3: pipewire-alsa type: plugin 4: pw-jack type: plugin
    tools: pactl,pw-cat,pw-cli,wpctl
Network:
  Device-1: MEDIATEK MT7921K Wi-Fi 6E 80MHz driver: mt7921e v: kernel modules: wl pcie: gen: 2
    speed: 5 GT/s lanes: 1 bus-ID: 29:00.0 chip-ID: 14c3:0608 class-ID: 0280
  IF: wlan0 state: up mac: <filter>
  Device-2: Realtek RTL8125 2.5GbE vendor: Micro-Star MSI driver: r8169 v: kernel pcie: gen: 2
    speed: 5 GT/s lanes: 1 port: f000 bus-ID: 2a:00.0 chip-ID: 10ec:8125 class-ID: 0200
  IF: eth0 state: down mac: <filter>
  IF-ID-1: wgpia0 state: unknown speed: N/A duplex: N/A mac: N/A
Bluetooth:
  Device-1: MediaTek Wireless_Device type: USB driver: btusb v: 0.8 bus-ID: 1-9:6
    chip-ID: 0e8d:0608 class-ID: e001 serial: <filter>
  Report: hciconfig ID: hci0 rfk-id: 0 state: up address: <filter> bt-v: 3.0 lmp-v: 5.2
    sub-v: 2305 hci-v: 5.2 rev: 2613
  Info: acl-mtu: 1021:6 sco-mtu: 240:8 link-policy: rswitch sniff link-mode: peripheral accept
    service-classes: rendering, capturing, object transfer, audio, telephony
Drives:
  Local Storage: total: 1.82 TiB used: 444.17 GiB (23.8%)
  SMART Message: Unable to run smartctl. Root privileges required.
  ID-1: /dev/nvme0n1 maj-min: 259:0 vendor: MSI model: M480 PRO 2TB size: 1.82 TiB block-size:
    physical: 512 B logical: 512 B speed: 63.2 Gb/s lanes: 4 type: SSD serial: <filter> rev: EIFM80.0
    temp: 43.9 C scheme: GPT
Partition:
  ID-1: / raw-size: 1.82 TiB size: 1.79 TiB (98.37%) used: 443.74 GiB (24.2%) fs: ext4
    dev: /dev/dm-0 maj-min: 253:0 mapped: luks-<filter>
  ID-2: /boot raw-size: 1024 MiB size: 973.4 MiB (95.06%) used: 439.3 MiB (45.1%) fs: ext4
    dev: /dev/nvme0n1p2 maj-min: 259:2
  ID-3: /boot/efi raw-size: 256 MiB size: 252 MiB (98.46%) used: 274 KiB (0.1%) fs: vfat
    dev: /dev/nvme0n1p1 maj-min: 259:1
Swap:
  Kernel: swappiness: 15 (default 60) cache-pressure: 100 (default)
  ID-1: swap-1 type: file size: 6 GiB used: 0 KiB (0.0%) priority: -2 file: /swap/swap
Sensors:
  System Temperatures: cpu: 49.9 C mobo: 46.0 C gpu: nvidia temp: 51 C
  Fan Speeds (RPM): N/A gpu: nvidia fan: 0%
Repos:
  Packages: 3421 pm: dpkg pkgs: 3392 libs: 1701 tools: apt,apt-get,aptitude,nala,synaptic pm: rpm
    pkgs: 0 pm: flatpak pkgs: 29
  No active apt repos in: /etc/apt/sources.list
  Active apt repos in: /etc/apt/sources.list.d/debian-stable-updates.list
    1: deb http://deb.debian.org/debian bookworm-updates main contrib non-free non-free-firmware
  Active apt repos in: /etc/apt/sources.list.d/debian.list
    1: deb http://deb.debian.org/debian bookworm main contrib non-free non-free-firmware
    2: deb http://security.debian.org/debian-security bookworm-security main contrib non-free non-free-firmware
  Active apt repos in: /etc/apt/sources.list.d/docker.list
    1: deb [arch=amd64] https://download.docker.com/linux/debian bookworm stable
  Active apt repos in: /etc/apt/sources.list.d/element-io.list
    1: deb [signed-by=/usr/share/keyrings/element-io-archive-keyring.gpg] https://packages.element.io/debian/ default main
  No active apt repos in: /etc/apt/sources.list.d/home:RizinOrg.list
  Active apt repos in: /etc/apt/sources.list.d/i2p.list
    1: deb [signed-by=/usr/share/keyrings/i2p-archive-keyring.gpg] https://deb.i2p.net/ bookworm main
  Active apt repos in: /etc/apt/sources.list.d/mx.list
    1: deb http://la.mxrepo.com/mx/repo/ bookworm main non-free
    2: deb http://la.mxrepo.com/mx/repo/ bookworm ahs
  Active apt repos in: /etc/apt/sources.list.d/signal-xenial.list
    1: deb [arch=amd64 signed-by=/usr/share/keyrings/signal-desktop-keyring.gpg] https://updates.signal.org/desktop/apt xenial main
  Active apt repos in: /etc/apt/sources.list.d/vscode.list
    1: deb [arch=amd64,arm64,armhf] https://packages.microsoft.com/repos/code stable main
  Active apt repos in: /etc/apt/sources.list.d/zulip-desktop.list
    1: deb https://download.zulip.com/desktop/apt stable main
Info:
  Processes: 618 Uptime: 3h 0m wakeups: 1 Memory: 31.27 GiB used: 8.41 GiB (26.9%) Init: SysVinit
  v: 3.06 runlevel: 5 default: graphical tool: systemctl Compilers: gcc: 12.2.0 alt: 12
  clang: 14.0.6 Client: shell wrapper v: 5.2.15-release inxi: 3.3.26
Boot Mode: UEFI
```

</p>
</details>

--- a/crates/gpui/Cargo.toml
+++ b/crates/gpui/Cargo.toml
@@ -79,8 +79,8 @@ futures.workspace = true
 gpui_macros.workspace = true
 http_client = { optional = true, workspace = true }
 image = "0.25.1"
+inventory = "0.3.19"
 itertools.workspace = true
-linkme = "0.3"
 log.workspace = true
 num_cpus = "1.13"
 parking = "2.0.0"

--- a/crates/gpui/src/action.rs
+++ b/crates/gpui/src/action.rs
@@ -173,7 +173,7 @@ struct ActionData {
 /// This type must be public so that our macros can build it in other crates.
 /// But this is an implementation detail and should not be used directly.
 #[doc(hidden)]
-pub type MacroActionBuilder = fn() -> MacroActionData;
+pub struct MacroActionBuilder(pub fn() -> MacroActionData);
 
 /// This type must be public so that our macros can build it in other crates.
 /// But this is an implementation detail and should not be used directly.
@@ -186,17 +186,13 @@ pub struct MacroActionData {
     pub json_schema: fn(&mut schemars::gen::SchemaGenerator) -> Option<schemars::schema::Schema>,
 }
 
-/// This constant must be public to be accessible from other crates.
-/// But its existence is an implementation detail and should not be used directly.
-#[doc(hidden)]
-#[linkme::distributed_slice]
-pub static __GPUI_ACTIONS: [MacroActionBuilder];
+inventory::collect!(MacroActionBuilder);
 
 impl ActionRegistry {
     /// Load all registered actions into the registry.
     pub(crate) fn load_actions(&mut self) {
-        for builder in __GPUI_ACTIONS {
-            let action = builder();
+        for builder in inventory::iter::<MacroActionBuilder> {
+            let action = builder.0();
             self.insert_action(action);
         }
     }

--- a/crates/gpui/src/gpui.rs
+++ b/crates/gpui/src/gpui.rs
@@ -104,7 +104,7 @@ mod window;
 #[doc(hidden)]
 pub mod private {
     pub use anyhow;
-    pub use linkme;
+    pub use inventory;
     pub use schemars;
     pub use serde;
     pub use serde_derive;

--- a/crates/gpui_macros/src/register_action.rs
+++ b/crates/gpui_macros/src/register_action.rs
@@ -13,9 +13,6 @@ pub fn register_action_macro(ident: TokenStream) -> TokenStream {
 }
 
 pub(crate) fn register_action(type_name: &Ident) -> proc_macro2::TokenStream {
-    let static_slice_name =
-        format_ident!("__GPUI_ACTIONS_{}", type_name.to_string().to_uppercase());
-
     let action_builder_fn_name = format_ident!(
         "__gpui_actions_builder_{}",
         type_name.to_string().to_lowercase()
@@ -38,10 +35,10 @@ pub(crate) fn register_action(type_name: &Ident) -> proc_macro2::TokenStream {
                         json_schema: <#type_name as gpui::Action>::action_json_schema,
                     }
                 }
-                #[doc(hidden)]
-                #[gpui::private::linkme::distributed_slice(gpui::__GPUI_ACTIONS)]
-                #[linkme(crate = gpui::private::linkme)]
-                static #static_slice_name: gpui::MacroActionBuilder = #action_builder_fn_name;
+
+                gpui::private::inventory::submit! {
+                    gpui::MacroActionBuilder(#action_builder_fn_name)
+                }
             }
         }
 

